packages/datagateway-download/cypress/integration/adminDownloadStatus.spec.ts

```diff
@@ -117,9 +117,14 @@ describe('Admin Download Status', () => {
 
     it('multiple columns', () => {
       cy.contains('[role="button"]', 'Username').click();
+      cy.get('.react-draggable')
+        .eq(4)
+        .trigger('mousedown')
+        .trigger('mousemove', { clientX: 600 })
+        .trigger('mouseup');
       cy.contains('[role="button"]', 'Access Method').click();
 
-      cy.get('[aria-rowindex="1"] [aria-colindex="3"]').should(
+      cy.get('[aria-rowindex="1"] [aria-colindex="5"]').should(
         'have.text',
         'globus'
```

Comment:
It might have just been the test data I was using but I plugged preprod data into this test and it came back with `'https'` instead of `'globus'`. Will it be `'https'` in the `bugfix/fix-e2e-tests-#712` branch? Just so I know if I can leave it or not

Reply:
All the branches I've been working on (local CI, fix e2e tests and this one) all expect globus on that test. These tests are difficult because you can't be too sure what has been downloaded beforehand. Good to know you tested it on preprod though, the tests need to be more robust for that than when I was using my local instance (with barely any downloads than the ones created for these tests).

Is it just the sorting on multiple columns that you had issues with on preprod?

Reply:
Yeah that was it. Otherwise I've tested everywhere else and it all seemed to be fine so I'm happy to approve :)

Reply:
I've just swapped round the button presses to make this test more robust (d8fbe0c27fa7d0a13776eed5cc3b7a5bfb57ecb7)